app/lib/common/toolkit/html_editor/mentions/mention_detection.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:acter/features/deep_linking/parse_acter_uri.dart';
+import 'package:acter/features/deep_linking/types.dart';
+import 'package:appflowy_editor/appflowy_editor.dart';
+
+UriParseResult? getMentionForInsert(TextInsert text) {
```

Comment:
same as before, just simplified and cleaned up a bit more for easier reusability